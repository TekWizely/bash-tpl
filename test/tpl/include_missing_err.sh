printf "%s\n" before
printf "%s\n" after
