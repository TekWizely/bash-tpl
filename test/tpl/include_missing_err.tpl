before
.INCLUDE not_found.tpl
after
