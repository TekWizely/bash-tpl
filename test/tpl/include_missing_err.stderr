File not found: 'not_found.tpl'
